src/pages/IPFSPin.vue

```diff
@@ -228,10 +238,26 @@ export default {
       cid_prompt: false,
       cid_input: '',
       cid_info: null,
-      cid_name: ''
+      cid_name: '',
+      selected_message: {
+        reason: 'None'
+      },
+      delete_prompt: false,
+      delete_field: ''
     }
   },
   methods: {
+    delete_rules (val) {
+      return new Promise((resolve, reject) => {
```

Comment:
What is this Promise actually doing? Why the timeout?

Reply:
Each time the user enters a character in the "DELETE" field a loading animation is played, this animation lasts 0.5 seconds

Reply:
I don't think that animation is necessary, just disabling the "delete" button seems enough. 
There is a "loading" animation, but this gives confusing feedback, as there is nothing to load really. 

Looking at how Github handles repo deletion for instance, there is no loading animation.

Reply:
I removed the animation